src/devices/src/virtio/queue.rs

```diff
@@ -26,29 +25,16 @@ pub(super) const VIRTQ_DESC_F_WRITE: u16 = 0x2;
 // The Virtio Spec 1.0 defines the alignment of VirtIO descriptor is 16 bytes,
 // which fulfills the explicit constraint of GuestMemoryMmap::read_obj_from_addr().
 
-#[derive(Debug, derive_more::From)]
+#[derive(Debug, derive_more::From, thiserror::Error)]
```

Comment:
Can `#[from]` from `thiserror::Error` be used? (then `derive_more::From` could be removed).

Reply:
Done.